xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-test/xwiki-platform-appwithinminutes-test-pageobjects/src/main/java/org/xwiki/appwithinminutes/test/po/ApplicationClassEditPage.java

```diff
@@ -97,14 +96,15 @@ public boolean hasPreviousStep()
      * Drags a field of the specified type from the field palette to the field canvas.
      *
      * @param fieldType the type of field to add, as displayed on the field palette
+     * @return the newly added field page object
      */
     public ClassFieldEditPane addField(String fieldType)
     {
         String fieldXPath = "//li[@class = 'field' and normalize-space(.) = '%s']";
         WebElement field = palette.findElement(By.xpath(String.format(fieldXPath, fieldType)));
-        // NOTE: We scroll the page up because the drag&drop fails sometimes if the dragged field and the canvas (drop
-        // target) are not fully visible. See https://code.google.com/p/selenium/issues/detail?id=3075 .
-        getDriver().createActions().sendKeys(Keys.HOME).perform();
+        // NOTE: We scroll to the top of the page because the drag&drop fails sometimes if the dragged field and the
+        // canvas (drop target) are not fully visible. See https://code.google.com/p/selenium/issues/detail?id=3075 .
+        scrollToTop();
```

Comment:
Would be good to provide more explanations. The issue seems to indicate it's related to chrome but I don't see any mention of that in the comment for ex.

Reply:
It would be good to explain also why we use JS  in scrollToTop().

Reply:
These are tricky stuff and we need the max info that we can review in the future.

Reply:
I've made a tentative documentation improvement in another PR https://github.com/xwiki/xwiki-platform/pull/1600